src/components/Schedule/AppointmentsPage.tsx

```
@@ -0,0 +1,273 @@
+      <div className="flex items-center gap-3 mb-4">
+        <h2 className="font-semibold capitalize text-base px-1">
+          {props.category.replace("-", " ")}
+        </h2>
+        <span className="bg-gray-200 px-2 py-1 rounded-md text-sm">
+          {data?.count ?? "..."}
+        </span>
+      </div>
+      <ScrollArea>
+        <ul className="space-y-3 px-0.5 pb-4 pt-1">
+          {appointments.map((appointment) => (
+            <li>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)